src/notify.ts

```diff
@@ -15,8 +15,8 @@ export const transports: { [key in TransportName]: TransportFn } = {
 export const transportNames = Object.keys(transports) as TransportName[];
 
 export enum EventType {
-  PROPOSAL_QUESTION_CREATED = "proposal_question_created",
-  NEW_ANSWER = "new_answer",
+  PROPOSAL_QUESTION_CREATED = "proposal-created",
```

Comment:
I found easier to change the enum's values to it matches the file name structure, instead of converting from snake to kebab case or having mixed naming conventions.